### PR TITLE
Fix typos in `op-conductor/README.md` documentation  

### PR DESCRIPTION
**Pull request by sky-coderay**
### Pull Request Description

**Description**  
This PR fixes minor typos in the `op-conductor/README.md` file:  
- Corrected the spelling of `showcaes` to `showcases` in the Architecture section.  
- Corrected `simple` to `simply` in the Leadership Transfer Process section.  

These changes improve the clarity and readability of the documentation.  

**Tests**  
No new tests were added as this is a documentation change. The updated file was manually reviewed for accuracy.  

**Additional context**  
These fixes ensure the documentation maintains a professional standard and avoids confusion for developers referencing it.

### CODE REVIEW

File: op-conductor/README.md [hunk 1]
```diff
@@ -93,6 +93,6 @@ There are 2 situations we need to consider.
 
 1. Leadership transfer triggered by raft consensus protocol (network partition, etc)
    1. In this case, a new leader will be elected regardless of its sync status, it could be behind for a few blocks
-   2. The solution is to simple, wait until the elected leader catch up to tip (same as the FSM tip)
+   2. The solution is to simply, wait until the elected leader catch up to tip (same as the FSM tip)
```
Comment (smartcontracts) [inline op-conductor/README.md]:
```suggestion
   2. The solution is to simply wait until the elected leader catch up to tip (same as the FSM tip)
```

Comment (sky-coderay) [reply op-conductor/README.md]: @smartcontracts  Thanks for catching that as well!

File: op-conductor/README.md [hunk 2]
```diff
@@ -93,6 +93,6 @@ There are 2 situations we need to consider.
 
 1. Leadership transfer triggered by raft consensus protocol (network partition, etc)
    1. In this case, a new leader will be elected regardless of its sync status, it could be behind for a few blocks
-   2. The solution is to simple, wait until the elected leader catch up to tip (same as the FSM tip)
+   2. The solution is to simply wait until the elected leader catch up to tip (same as the FSM tip)
```
Comment (smartcontracts) [inline op-conductor/README.md]:
```suggestion
   2. The solution is to simply wait until the elected leader catches up to tip (same as the FSM tip)
```

Comment (sky-coderay) [reply op-conductor/README.md]: Done !

**Review by zhwrd — APPROVED** [review]

**Pull request merged**